anndata/_core/aligned_mapping.py

```diff
@@ -277,6 +276,10 @@ def __init__(
         self.subset_idx = subset_idx
         self._axis = parent_mapping._axis
 
+    @property
```

Comment:
This should probably be defined on the AlignedViewMixin instead of on two subclasses, right?

Reply:
true, that’s even better!